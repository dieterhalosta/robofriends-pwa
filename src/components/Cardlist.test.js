import { shallow } from "enzyme";
import React from "react";
import CardList from "./CardList";

it("expect to render Card Component", () => {
  const mockRobots = [
    {
      id: 1,
      name: "John Snow",
      username: "JohnJohn",
      email: "john@gmail.com"
    }
  ];
  const cardComponent = shallow(<CardList robots={mockRobots} />);
  expect(cardComponent.debug()).toMatchSnapshot();
});
