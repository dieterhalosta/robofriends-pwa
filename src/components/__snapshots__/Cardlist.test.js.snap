// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render Card Component 1`] = `
"<div>
  <Card id={1} name=\\"John Snow\\" email=\\"john@gmail.com\\" />
</div>"
`;
